tensorflow/tools/pip_package/setup.py

```diff
@@ -55,13 +55,6 @@
   # mock comes with unittest.mock for python3, need to install for python2
   REQUIRED_PACKAGES.append('mock >= 2.0.0')
 
-# remove tensorboard from tf-nightly packages
-if 'tf_nightly' in project_name:
-  for package in REQUIRED_PACKAGES:
-    if 'tensorflow-tensorboard' in package:
-      REQUIRED_PACKAGES.remove(package)
```

Comment:
Should we keep this block, and add tb-nightly if the project name is tf-nightly?

Reply:
Whoops. I think it might be best if the `master` branch tracked `tb-nightly` in general. Then I'll send a PR for each new release branch, updating it to point to the tensorflow-tensorboard package.

Reply:
I think it would be better if we could have the release process as self-contained as possible.
The current use of `build_pip_package` script is the default with no args will build for main TF package, and for nightlies we set the project name differently.
I prefer we conformed to that.

Reply:
We could do that, but would need to remove the version range because no `tensorflow-tensorboard` packages exist within `1.5.0a0, < 1.6.0a0`. What would you have me do?

Reply:
I see. Then for master branch, can we have the dependency without the range, and in the replacement here we add the tb-nightly dependency with the range?

Reply:
Can we switch between tb-nightly and tensorflow-tensorboard depending on the project_name?

Reply:
Your wish is my command. Done.